v2/samples/network/v1api20241001/v1api20240301_networksecuritygroup.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: network.azure.com/v1api20240301
```

Comment:
This sample shouldn't be here? Wrong API version?

Reply:
This NSG is what the FlowLog is "watching", so it's needed. I'll move it into the refs folder.